src/bin/casc.rs

```diff
@@ -35,3 +37,32 @@ fn main() -> std::io::Result<()> {
         Ok(s) => out_file.write_all(s.as_bytes()),
     }
 }
+
+// Create a list of policy files
+// Display info and proceed on errors.  As long as we have at least one file, errors should be
+// considered non-fatal
+fn get_policy_files(filenames: Vec<String>) -> Vec<String> {
+    let mut policy_files = Vec::new();
+    for file in filenames {
+        for entry in WalkDir::new(file) {
+            match entry {
+                Ok(entry) => {
+                    if entry.file_type().is_file()
+                        && entry.path().extension() == Some(OsStr::new("cas"))
```

Comment:
`.contains()` would be perfect but since it is nightly-only, you can use ` entry.path().extension().unwrap_or_default() == "cas"`.

Reply:
Updated.  Thanks.